lib/export_task.rb

```diff
@@ -24,6 +24,13 @@ def export(output_dir, options)
     @exporter.system_description.assert_scopes("os")
     @exporter.system_description.validate_export_compatibility
 
+    ["unmanaged_files", "changed_managed_files", "config_files"].each do |scope|
+      if @exporter.system_description[scope] &&
+        !@exporter.system_description.scope_extracted?(scope)
+          raise Machinery::Errors::ExportFailed.new("No extracted files in scope #{scope}")
```

Comment:
There's already a `MissingExtractedFiles` exception class which shows some help about how to extract the missing files, maybe you could also use it here.